pkg/backend/httpstate/client/client.go

```diff
@@ -1127,6 +1127,30 @@ func (pc *Client) UpdateStackTags(
 	return pc.restCall(ctx, "PATCH", getStackPath(stack, "tags"), nil, tags, nil)
 }
 
+func (pc *Client) UpdateStackDeployment(ctx context.Context, stack StackIdentifier,
+	deployment apitype.DeploymentSettings,
+) error {
+	// TODO(german): validate deployment
+
+	return pc.restCall(ctx, "POST", getStackPath(stack, "deployments", "settings"), nil, deployment, nil)
+}
+
+func (pc *Client) DestroyStackDeployment(ctx context.Context, stack StackIdentifier) error {
+	// TODO(german): validate deployment
```

Comment:
It doesn't look like we have a deployment to validate here?  Is this TODO accurate?